java/ql/src/Security/CWE/CWE-489/DebuggableAttributeEnabled.qhelp

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>When a debugger is enabled it could allow for entry points in the application or reveal sensitive information.</p>
```

Comment:
I think this could use a first sentence that introduces the concept of the manifest file in Android apps and when/why the `debuggable` attribute is normally used, before jumping straight into the problem the query is trying to model.

You can take inspiration from the first sentence in the `overview` section of [this example](https://github.com/github/codeql/blob/main/docs/query-help-style-guide.md#query-help-example).

Reply:
Updated in commit [17d99ff](https://github.com/github/codeql/pull/9939/commits/17d99ff86c2d4d3f754e49dc9342e7775fc6cc0f). 
@atorralba Let me know if you think any additional edits are needed.

Reply:
Looks much better, thanks!